test/Microsoft.AspNet.Mvc.TagHelpers.Test/LinkTagHelperTest.cs

```diff
@@ -125,12 +118,14 @@ public static TheoryData MultiAttributeSameNameData
                 FallbackTestValue = "hidden",
                 Href = "test.css",
             };
+            var expectedAttributes = output.Attributes;
+            expectedAttributes.Add(new TagHelperAttribute("href", "test.css"));
```

Comment:
Do not change the `output.Attributes` collection. Instead use `expectedAttributes = new TagHelperAttributeList(output.Attributes)`.